SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialActorChannel.cpp

```diff
@@ -1192,19 +1189,25 @@ void USpatialActorChannel::OnCreateEntityResponse(const Worker_CreateEntityRespo
 	switch (static_cast<Worker_StatusCode>(Op.status_code))
 	{
 	case WORKER_STATUS_CODE_SUCCESS:
-		UE_LOG(LogSpatialActorChannel, Verbose, TEXT("Create entity request succeeded. "
-			"Actor %s, request id: %d, entity id: %lld, message: %s"), *Actor->GetName(), Op.request_id, Op.entity_id, UTF8_TO_TCHAR(Op.message));
+		UE_LOG(LogSpatialActorChannel, Verbose,
+			   TEXT("Create entity request succeeded. "
```

Comment:
The indentation of 3 spaces here seems off - can we keep it to tabs?

Reply:
Im really confused why clang keeps doing this

Reply:
found it

Reply:
So the code here is actually a correct alignment of with the open bracket ( above but Github is displaying it weirdly. The DontAlign alternative actually seems worse I think - it pushes the first argument after the ( onto the next line.